src/Microsoft.AspNet.Session/SessionMiddleware.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Microsoft.AspNet.Session
 {
+    /// <summary>
+    /// Enables the session state for the application.
+    /// </summary>
```

Comment:
You are right, nothing but we should have a rich docs for all the public APIs, really i impressed by what @DamianEdwards did in **Localization** repo, because he provide a rich docs for all the public API's if i'm not wrong